go.mod

```diff
@@ -1,3 +1,5 @@
 module github.com/fluidpay/iso8583
 
 go 1.14
+
+require github.com/stretchr/testify v1.6.1
```

Comment:
If you could just use a self-implemented equals function instead of the testify that would be nice.